### ISSUE TITLE
Git LFS version in Microsoft Git installer is much older than version in original installer

### ISSUE THREAD

**Issue by nikola-sh** [microsoft/git#278]
Git for Windows installer contains [Git LFS](https://github.com/git-lfs/git-lfs).

Git LFS version for Microsoft package is much older than original Git for Windows:
- Git 2.27 use Git LFS 2.11.0
- MS Git 2.27 use Git LFS 2.7.2

**Comment by derrickstolee** [comment]
I'm surprised that Git LFS is packaged at all. I thought it wasn't. I see now that I simply had not checked the correct box in the installer wizard:

![image](https://user-images.githubusercontent.com/570044/85167232-7f2f3600-b236-11ea-9f55-6359ddd7fee0.png)

Thanks for bringing this to our attention. We don't want to have users using old versions.

**Comment by dscho** [comment]
I know what the problem is: we never upgraded to the newer MSYS2 Runtime. See [the logs](https://dev.azure.com/gvfs/ci/_build/results?buildId=18799&view=logs&j=00af5505-930d-50dd-3844-b167e8b4bb13&t=3ffc0b05-3b5b-52e7-d7a9-341a18b33b5d&l=17):

```
From https://github.com/git-for-windows/git-sdk-64
 * branch              msys2-runtime-2.x -> FETCH_HEAD
```

**Comment by derrickstolee** [comment]
Fixed in v2.27.0.vfs.1.0. Thanks for the report!
